src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -90,6 +99,21 @@ public function theStoreHasAPaymentMethodWithACode($paymentMethodName, $paymentM
     }
 
     /**
+     * @Given /^(this payment method) is named "([^"]+)" in "([^"]+)"$/
+     */
+    public function thisPaymentMethodIsNamedIn(PaymentMethodInterface $paymentMethod, $name, $locale)
+    {
+        /** @var PaymentMethodTranslationInterface $translation */
+        $translation = $this->paymentMethodTranslationFactory->createNew();
+        $translation->setName($name);
+        $translation->setLocale($locale);
+
+        $paymentMethod->addTranslation($translation);
```

Comment:
Unfortunately doesn't work.
Either it has something to do with the behat scope, or it's a possible bug.